source/data-api/data-formats.txt

```diff
@@ -67,6 +60,42 @@ endpoints and individually for each custom endpoint. Incoming requests
 can also specify a preferred data format that overrides the default
 using an ``Accept`` header.
 
+.. _data-format-json:
+
+Standard JSON (application/json)
+--------------------------------
+
+The **JSON** format uses standard types that any tool can parse and
+understand. However, JSON cannot represent all BSON types so JSON
+responses may lose type information for some fields. For example, BSON
+has distinct types for 32-bit integers and 64-bit floats but a JSON
+response represents both as a ``number``.
+
+.. _data-format-ejson:
+
+Canonical & Relaxed EJSON (application/ejson)
+---------------------------------------------
+
+The **EJSON** format, short for :manual:`MongoDB Extended JSON
+</reference/mongodb-extended-json/>`, is a superset of standard JSON
+that uses structured fields to represent BSON data that don't have
+corresponding JSON types. This fully represents your data but requires
+your client to understand how to work with EJSON.
+
+There are two variants of EJSON:
+
+- **Canonical EJSON** uses a verbose structure that emphasizes type
+  preservation at the expense of readability and interoperability. It
```

Comment:
How is it at the expense of interoperability per se? Just wondering if this is totally accurate.

Reply:
Copied directly from https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/ - I think it's referring to how JS has a built in `JSON.parse()` but not `EJSON.parse()`.